src/NServiceBus.Core.Analyzer.Tests/UniformSessionStubs/IUniformSession.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.UniformSession
+{
+    using System;
+    using System.Threading.Tasks;
+
+    public interface IUniformSession
```

Comment:
Is there any difference in the API surface area for the NSB 6 version of the UniformSession package?

Reply:
no, the APIs are exactly the same.